README.md

```diff
@@ -28,7 +28,47 @@
 //Todo: list general conventions for development
 
 //Todo: helpful information and how-to's for development
-
+## Possible fix testcontainers when using wsl instead of Docker Desktop
+**This fix needs certain modifications in the App.cs. 
+When working with other team-members who are on other platforms or use Docker Desktop instead of wls this fix will break their setup**
```

Comment:
```suggestion
When working with other team members who are on other platforms or use Docker Desktop instead of WSL this fix will break their setup.**
```